_projects/model-my-watershed.md

```diff
@@ -0,0 +1,26 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: project
+
+# NO EDITING ABOVE THIS LINE
+# # # # # #
+
+seo_description: "GeoTrellis runs the hydrologic models and returns results to students as they make simulated changes to the landscape in the Model My Watershed application, created with the Stroud Water Resource Center."
+title: Model My Watershed
+featured-image: ./assets/images/feature_model-my-watershed.jpg
+featured-image-alt: Screenshot of an outlined watershed in Model My Watershed.
+page_class: l-project
+order: 1
+
+# # # # # #
+# Page copy
+# # # # # #
+intro: The Stroud Water Resource Center received funding from the National Science Foundation to create new educational tools that will help students learn by interacting with watershed processes in a game-like environment.
+body: GeoTrellis runs hydrologic models and quickly returns results as student users adjust parameters in Model My Watershed. This allows them to simulate changes to the landscape and then visualize the impact of these changes on the local ecosystem. Students are also able to draw custom bounding boxes and query against several raster datasets at web speed. Read more about the project [here.](https://wikiwatershed.org/documentation/mmw-tech/)
```

Comment:
' at web speed' I think we can strike this (seems like a show, don't tell sort of thing)